src/test/java/cloudBankTestCases/ClientCreationTestCase.java

```diff
@@ -0,0 +1,99 @@
+package cloudBankTestCases;
+
+import cloudBankPages.ClientListPage;
+import cloudBankPages.CreateClientPage;
+import cloudBankPages.Homepage;
+import cloudBankPages.LogInPage;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+import java.time.Duration;
+
+import static org.testng.Assert.assertTrue;
+
+public class ClientCreationTestCase {
+
+    WebDriver driver;
+
+    LogInPage objLogInPage;

```

Comment:
Do you really need the "obj" prefix? For me, it seems unnecessesary.

Reply:
updated the object name